src/main/flight/pid.c

```diff
@@ -1127,6 +1145,17 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
             if (cmpTimeUs(currentTimeUs, levelModeStartTimeUs) > CRASH_RECOVERY_DETECTION_DELAY_US) {
                 detectAndSetCrashRecovery(pidProfile->crash_recovery, axis, currentTimeUs, delta, errorRate);
             }
+            // log the roll values only, since we can't log them all, for testing
+            if (axis == FD_ROLL) {
```

Comment:
In the CLI you can choose which axis you want to log
```suggestion
            if (axis == gyro.gyroDebugAxis) {
```

Reply:
I keep forgetting that.
But now I think we just use the AccDelta / Jerk values.
From checking logs, the gyro values don't add much, take CPU, and increase flash space.
We can probably improve CrashRecovery by using only AccDelta